tests/fixtures/saas/hubspot_fixtures.py

```diff
@@ -137,8 +137,9 @@ def hubspot_erasure_data(connection_config_hubspot, hubspot_erasure_identity_ema
     # no need to subscribe contact, since creating a contact auto-subscribes them
 
     # Allows contact to be propagated in Hubspot before calling access / erasure requests
-    remaining_tries = 5
+    remaining_tries = 10
```

Comment:
I wonder if we could encapsulate these things in config vars specific to these tests so they can be tuned easier. Do we have a clear picture of how long a contact takes to propagate through Hubspot, or does it vary?

Reply:
It varies but this is a good future proposal to have the configuration in one place